spec/features/features_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'takeaway'
+require 'menu'
+
+  describe 'feature tests' do
+
+    it 'allows a customer to view the available dishes with prices' do
+      # As a customer
+      # So that I can check if I want to order something
+      # I would like to see a list of dishes with prices
+      m = Menu.new
+      t = Takeaway.new(m)
+      expect(t.show_menu).to include("Spam")
+    end
+
+    it 'allows a customer to select from the available dishes' do
+      # As a customer
+      # So that I can order the meal I want
+      # I would like to be able to select some number of several available dishes
```

Comment:
Metrics/LineLength: Line is too long. [81/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)